src/modules/likeController.js

```diff
@@ -0,0 +1,33 @@
+import * as Variable from './globalVar.js';
+
+const likeMoov = async (id) => {
+  await fetch(`${Variable.involvementApiUrl}likes`, {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: id,
+    }),
+    headers: { 'Content-type': 'application/json; charset=UTF-8' },
+  });
+};
+
+const likesReader = async () => {
+  await fetch(`${Variable.involvementApiUrl}likes/`)
+    .then((response) => response.json())
+    .then((json) => {
+      document.querySelectorAll('.likes-number').forEach((itemLike, index) => {
+        itemLike.textContent = (json[index].likes > 1) ? (`${json[index].likes} Likes`) : (`${json[index].likes} Like`);
+      });
+    });
+};
```

Comment:
As per the project requirements, when a user clicks on the Like button of an item, the interaction should get recorded in the involvement API, and the screen gets updated. However, in your case, the former happens but the latter doesn't take effect immediately. I either have to refresh the page or click the like button again to see the number of like getting updated. But when I click the like button again, it displays the like number of previous clicked event. And when I refresh the page, it displays the correct like number. Kindly fix this error.

https://github.com/Graciano1997/js_moduleII_capstone/assets/33268927/9e8df8e5-6898-4f99-91ba-941b921310c7

Reply:
hello @muneebulrehman  .